projects/bsquared/index.js

```diff
@@ -1,49 +1,84 @@
 const sdk = require('@defillama/sdk');
-const { sumTokensExport: sumBRC20TokensExport } = require("../helper/chain/brc20");
-const { sumTokensExport } = require('../helper/sumTokens');
+const {sumTokensExport: sumBRC20TokensExport} = require("../helper/chain/brc20");
+const {sumTokensExport} = require('../helper/sumTokens');
 const ADDRESSES = require('../helper/coreAssets.json')
 
-const BTCOwners = [
-  "bc1q4fsk5pgnmnu7ngp79xclsw2t0yk46sjqw22ffz",
-  "bc1qva5m5e7da2zm590z03tdnj42u9q2uye3hgrehwrzgg8j4kxq9seq9rvw0m",
-  "bc1qjv2lfrv672rqagycs5zdsggmury0cz2vufek46jj86ddqynyp2qsxm3qfs",
-];
-const BRC20Owners = [
-  "bc1q97ctqygjgj0ljxgge4q735ujxvlad8smass7f0axc6x3ggffr8xqwn69hc",
+// const BTCOwners = [
+//     "bc1q4fsk5pgnmnu7ngp79xclsw2t0yk46sjqw22ffz",
+//     "bc1qva5m5e7da2zm590z03tdnj42u9q2uye3hgrehwrzgg8j4kxq9seq9rvw0m",
+//     "bc1qjv2lfrv672rqagycs5zdsggmury0cz2vufek46jj86ddqynyp2qsxm3qfs",
+// ];
+// const BRC20Owners = [
+//     "bc1q97ctqygjgj0ljxgge4q735ujxvlad8smass7f0axc6x3ggffr8xqwn69hc",
+// ];
+
+// const ADDRESSES_ETHEREUM_STONE = '0x7122985656e38BDC0302Db86685bb972b145bD3C';
+
+const ADDRESSES_BSQUARED_BSTONE = '0x7537C1F80c9E157ED7AFD93a494be3e1f04f1462';
+const ADDRESSES_BSQUARED_FDUSD = '0xC2Fe4f673455Ef92299770a09CDB5E8756A525D5';
+const ADDRESSES_BSQUARED_MATIC = '0xc3ee2Df14B1Bc526c24ED802f1873d49664a0d5c';
+const ADDRESSES_BSQUARED_ORDI = '0xa0f4470B714677AEEcE0d20074c540b3Cf6a477E';
+const ADDRESSES_BSQUARED_SATS = '0x7eBFcE05E418C380a2b6EB0F65995cA04ef4bc00';
+
+const tokenList = [
+    ADDRESSES.null,
+    ADDRESSES.bsquared.USDT,
+    ADDRESSES.bsquared.USDC,
+    ADDRESSES.bsquared.WBTC,
+    ADDRESSES_BSQUARED_BSTONE,
+    ADDRESSES_BSQUARED_FDUSD,
+    ADDRESSES_BSQUARED_MATIC,
+    ADDRESSES_BSQUARED_ORDI,
+    ADDRESSES_BSQUARED_SATS,
 ];
 
-const ADDRESSES_ETHEREUM_STONE = '0x7122985656e38BDC0302Db86685bb972b145bD3C';
+const ADDRESS_UNLOCK = '0x463d0908ba1F21B791883a6c27Aa82d293Dd0f69';
+const ADDRESS_LOCK = '0x9683826a04db8ae2e256e6a14b87d440c7105824';
```

Comment:
sorry, it is not very clear, what are these contracts? second one seems to be an EOA

Reply:
> sorry, it is not very clear, what are these contracts? second one seems to be an EOA


1) When the user is stake, the asset is transferred to the address_lock address;
2) When the user is unstake, the asset will be transferred from the address_unlock address;
3) Periodically check and transfer assets from address_lock to address_unlock;

Reply:
Do you want to provide that the monitored address is the contract address?